ci/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM google/cloud-sdk
+ARG HELM_VERSION=3.1.3
+ARG KUBECTL_VERSION=1.16.9
+
+RUN mkdir -p /src
+WORKDIR /src
+
+# Install Docker client
+RUN apt-get update && \
+    apt-get install -y apt-transport-https \
+                       ca-certificates \
+                       curl \
+                       gnupg2 \
+                       software-properties-common \
+                       wget && \
+    curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg | apt-key add - && \
```

Comment:
Download/save the key and then import rather than piping - it doesn't generate warnings and is less fragile

Reply:
Fixed.